### DISCUSSION TITLE
Synchronize Interactions in Multi-Window Plot

### DISCUSSION THREAD

**Discussion by lucasgoddamn** [pyvista/pyvista#3315] [Q&A] [answered]
Hello,
im trying to compare multiple 3D NIFTI images by putting them next to each other in a Multi-Window Plot.
My issue is that i cant find any way to synchronize interactions like Camera movement or Threshhold adjustments. In the Camera and Multi-Window Plot Docs it doesn't seem to be a thing. 
Is there a way to solve my issue?
Thank you a lot in advance!

**Comment by banesullivan** [comment, marked as answer]
Perhaps you're looking for the `link_views()` method? Ref: https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Plotter.link_views.html?highlight=link_views#pyvista.Plotter.link_views

**Reply by lucasgoddamn** [reply]
This is exactly what i wanted! Thank you a lot.
